packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.less

```diff
@@ -0,0 +1,5 @@
+@import (reference) "node_modules/@nova-ui/bits/sdk/less/nui-framework-variables";
+
+.description-primary {
```

Comment:
Is this a nova-charts class? If yes, then I'd prefer adding our own class with a BEM compatible name to avoid styling a generic class.